app/javascript/packs/provider_active_docs.ts

```diff
@@ -0,0 +1,20 @@
+// We can define the 3scale plugins here and export the modified bundle
+import 'swagger-ui/dist/swagger-ui.css'
+
+import { renderApiDocs } from 'ActiveDocs/ThreeScaleApiDocs'
+
+import 'ActiveDocs/swagger-ui-3-provider-patch.scss'
+
+// eslint-disable-next-line @typescript-eslint/no-misused-promises
+document.addEventListener('DOMContentLoaded', async () => {
+  const containerId = 'api-containers'
+  const container = document.getElementById(containerId)
+
+  if (!container) {
+    throw new Error('The target ID was not found: ' + containerId)
+  }
+
+  const { baseUrl = '', apiDocsPath = '', apiDocsAccountDataPath = '' } = container.dataset
+
+  await renderApiDocs(container, apiDocsPath, baseUrl, apiDocsAccountDataPath)
```

Comment:
This is not awaiting anything, so it's better to explicitly ignore promise:
```suggestion
  void renderApiDocs(container, apiDocsPath, baseUrl, apiDocsAccountDataPath)
```
No need to make this block async and disable the eslint rule.

===
Ideally we should have here a loading/loaded/fail mechanism, and we would use `then` and `catch` and the promise wouldn't have to be ignore. But since we don't have that, let's just ignore it 😄

Reply:
I assumed that as `renderApiDocs` itself is async (as it waits for an async function inside), we needed to wait for its also.